dspace-api/pom.xml

```diff
@@ -828,6 +828,22 @@
             <version>2.0.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
```

Comment:
As we've discussed, try getting rid of this dependency in favor of some json processing library already present in dspace